app/services/idv/steps/in_person/ssn_step.rb

```diff
@@ -24,7 +23,8 @@ def call
 
           idv_session.delete('applicant')
 
-          maybe_redirect_to_verify_info
+          flow_session[:flow_path] = @flow.flow_path
```

Comment:
What is this doing?

Reply:
It grabs the flow_path ('hybrid' or 'standard' for DocumentCapture) from the Flow and puts it into the flow_session where non-FSM controllers can get to it. That value is expected in analytics.

Reply:
It was evaluating to null so the 'in person path' spec in `analytics_spec.rb` failed

![Screenshot 2023-06-28 at 8 32 06 AM](https://github.com/18F/identity-idp/assets/125507397/0e71a3d7-5f8e-4431-84fe-d077354e7765)

Reply:
Thanks! Not blocking but would love to see a comment explaining what it's doing
```suggestion
          # save the flow path so it's saved to analytics, eg standard or hybrid flow
          flow_session[:flow_path] = @flow.flow_path
```

Reply:
Done! Resolved with commit 8d155c32d76117b5c6aeb1312c7c6ef24d88af92

Reply:
FYI, Team Ada has ticket [LG-9335](https://cm-jira.usa.gov/browse/LG-9335) to move flow_path out of the flow_session into IdvSession. It might have to wait until you folks are done with your steps, though. Or maybe they're independent since you have your own flow_session.